9.4/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -235,6 +235,45 @@ enough space for the copy;  upgrade failure because of not enough space might
 lead to data loss.
 
 
+Extending image
+----------------
+
+This image can be extended using
+[source-to-image](https://github.com/openshift/source-to-image).
+
+For example to build customized image `new-postgresql`
+with configuration in `~/image-configuration/` run:
+
+
+```
+$ s2i build ~/image-configuration/ postgresql new-postgresql
+```
+
+The directory passed to `s2i build` should contain one or more of the
+following directories:
+
+##### `postgresql-config/`
+
+contained files will be included at the end of image postgresql.conf file during database initialization
+
+
+##### `init/`
+
+contained shell scripts (`*.sh`) are sourced once, when database is initialized
+
+##### `start-hook/`
```

Comment:
Still no comment about `s/start-hook/postgresql-start-hook/`.  Mariadb has `mysql-cfg`, `mysql-init` and `mysql-pre-init`.  Shouldn't we have the namespaced hook directory, too?